app/service/sorting.go

```diff
@@ -42,13 +42,11 @@ func (d sortingDirection) conditionSign() string {
 	return "<"
 }
 
-const idFieldName = "id"
-
 // ApplySortingAndPaging applies ordering and paging according to given accepted fields and sorting rules
 // taking into the account the URL parameters 'from.*'
 func ApplySortingAndPaging(r *http.Request, query *database.DB, acceptedFields map[string]*FieldSortingParams,
-	defaultRules string) (*database.DB, APIError) {
-	sortingRules, acceptedFields := prepareSortingRulesAndAcceptedFields(r, acceptedFields, defaultRules)
+	defaultRules string, skipSortParameter bool) (*database.DB, APIError) {
```

Comment:
Reading the definition + spec of the function, I cannot understand what this `skipSortParameter`is for.

Reply:
added the comment